google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/SelectConfigDestinationFolderDialog.java

```diff
@@ -53,6 +55,25 @@ public SelectConfigDestinationFolderDialog(@Nullable Project project) {
         project,
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
+
+    if (project != null && project.getBasePath() != null) {
+      String srcPath = project.getBasePath() + "/src";
+      String srcMainPath = project.getBasePath() + "/src/main";
+      String AppenginePath = project.getBasePath() + "/src/main/appengine";
+      String dockerPath = project.getBasePath() + "/src/main/docker";
+
+      if (fileType == ConfigFileType.APP_YAML && new File(AppenginePath).isDirectory()) {
```

Comment:
This simplifies things. Now presenting canonical paths all the time. Even for a non-existing directories, users will be able to create them.